test/local-time.js

```diff
@@ -66,4 +66,12 @@ suite('local-time', function() {
     }
     assert.include(['1969', '1970'], root.children[0].textContent)
   })
+
+  ;('Intl' in window ? test : test.skip)('displays time zone name', function() {
+    var time = document.createElement('local-time')
+    time.setAttribute('datetime', '1970-01-01T00:00:00.000Z')
+    time.setAttribute('minute', '2-digit')
+    time.setAttribute('timeZoneName', 'short')
```

Comment:
I suppose it is strange to have a mixed case attribute name when all others are lowercase. I'm fine with `<local-time time-zone-name="">` or `<local-time timezonename="">`. The property would still be camel cased as `timeZoneName` to match the formatter.